product/dialogs/miq_dialogs/miq_provision_openstack_dialogs_template.yaml

```diff
@@ -164,6 +164,15 @@
           :required: true
           :display: :edit
           :data_type: :integer
+        :cloud_network_choice_type:
```

Comment:
Suggest using `cloud_network_type`.  Since these are dialog options aren't all fields chosen "choice" values.

Reply:
cloud_network_type would be misleading. What is being selected here is not a type of network, but the method of selecting the network. I'm open to other words, though, if you think choice is confusing. cloud_network_identification_method perhaps?

Reply:
My suggestion above is a bit too long. I'm going to change it to cloud_network_selection_method when I move it into the openstack repo.